README.md

```diff
@@ -184,13 +184,17 @@ syslog-ng packages are released for the following distribution versions (x86-64)
     wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | sudo apt-key add -
     ```
 
-2. Add the repository containing the latest build of syslog-ng to the APT sources. For example, stable releases on Ubuntu 22.04:
+2. Add the repository containing the latest build of syslog-ng to the APT sources. For example, stable releases on Ubuntu 24.04:
 
     ``` shell
-    echo "deb https://ose-repo.syslog-ng.com/apt/ stable ubuntu-jammy" | sudo tee -a /etc/apt/sources.list.d/syslog-ng-ose.list
+    echo "deb https://ose-repo.syslog-ng.com/apt/ stable ubuntu-noble" | sudo tee -a /etc/apt/sources.list.d/syslog-ng-ose.list
```

Comment:
it might be worth adding a note about the per-version differences as a comment, like

`# Replace `noble` with your current version accordingly`

or s.g. like that

Reply:
Updated with requested change.

Reply:
Please see line 188

> For example if you are running Ubuntu 24.04, you would use `ubuntu-noble`, see chart above: